packages/block-editor/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## Master
+
+### Deprecation
```

Comment:
Not sure what this change is doing here 🤔

Reply:
Looks like we're fine, this is part of https://github.com/WordPress/gutenberg/pull/17195/files

Reply:
Yes, this PR landed yesterday 👍